server/cars.js

```diff
@@ -75,13 +117,60 @@ router.get('/:carId', auth.verifyToken, User.verify, async (req, res) => {
     let car = await Car.findOne({
       _id: req.params.carId
     });
+
+    const nextOil = getNextService(car.mileage, intervals.OIL, car.oil);
+    const nextTransmission = getNextService(car.mileage, intervals.TRANSMISSION,car.transmission);
+    const nextTire = getNextService(car.mileage, intervals.TIRE_ROTATION, car.tire);
+    const nextCoolant = getNextService(car.mileage, intervals.COOLANT, car.coolant);
+    const nextAir = getNextService(car.mileage, intervals.AIR_FILTER, car.engine);
+
+    car._doc.maintenanceItems = [
+      {
+        title: 'Oil Change',
+        next: nextOil,
```

Comment:
I changed each object to use `next` as the key word so it'll make it easier in vue. You can go {{ item.next }}